src/rai/rai/utils/configurator.py

```
@@ -0,0 +1,528 @@
+            if not os.getenv("LANGFUSE_SECRET_KEY", "") or not os.getenv(
+                "LANGFUSE_PUBLIC_KEY", ""
+            ):
+                success = False
+                st.error(
+                    "Langfuse is enabled but LANGFUSE_SECRET_KEY or LANGFUSE_PUBLIC_KEY is not set"
+                )
+
+        progress.progress(0.5)
+        use_langsmith = st.session_state.config["tracing"]["langsmith"]["use_langsmith"]
+        if use_langsmith:
+            if not os.getenv("LANGCHAIN_API_KEY", ""):
```

Ruff

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)